### ISSUE TITLE
Batch download

### ISSUE THREAD

**Issue by hclivess** [setnicka/ulozto-downloader#123]
Is there a way to add more than 1 file to download (in a queue)?

**Comment by Sperhak323** [comment]
Yes i write script where you can download 2 file in a queue. See here https://github.com/Sperhak323/ulozto-downloader-script

**Comment by setnicka** [comment]
I will reopen this issue as feature request. This should be implemented easily in the ulozto-downloader itself when you give it multiple URLs.

**Comment by hclivess** [comment]
Thanks. I made a fork and I'm running few instances in parallel (manually), which is simpler because I fill out all the captchas at start. I added support for manual restart using batch files (windows). I also added name of the download to the captcha window so I know which captcha belongs to which download (I'm too lazy to downgrade Python for ml detection). If you find use for any of those changes, they're in my fork.
